src/modules/chuck-norris/api/chuck-norris-routes.ts

```diff
@@ -0,0 +1,178 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import { getQtyFromRequest } from '../../../utils/route-utils';
+import ColorErrors from '../consts/chuck-norris-errors';
+import facts from "./chuckfacts.json"
+/**
+ * @openapi
+ * definitions:
+ *   ChuckNorrisFact:
+ *     description: Chuck Norris Facts were obtained from api.chucknorris.io
+ *     type: object
+ *     properties:
+ *       categories:
+ *         type: array
+ *         items:
+ *           type: string
+ *         example: ["science","dev"]
+ *         description: A list of categories that this fact relates to.
+ *       icon:
+ *         type: string
+ *         example: https://assets.chucknorris.host/img/avatar/chuck-norris.png
+ *         description: URL for an icon - provided in the original data from api.chucknorris.com      
+ *       id:
+ *         type: string
+ *         example: izjeqnjzteeqms8l8xgdhw
+ *         description: ID from api.chucknorris.io
+ *       sourceUrl:
+ *         type: string
+ *         example: https://api.chucknorris.io/jokes/izjeqnjzteeqms8l8xgdhw
+ *         description: URL for this Fact on api.chucknorris.io
+ *       value:
+ *         type: string
+ *         example: Chuck Norris knows the last digit of pi.
+ */
+
+/**
+ * @openapi
```

Comment:
Hey, this seems to be a duplicate of the comment above :)

Reply:
confirmed - have removed the duplicate